source/material_model/rheology/elasticity.cc

```diff
@@ -146,8 +147,27 @@ namespace aspect
         if (this->convert_output_to_years())
           fixed_elastic_time_step *= year_in_seconds;
 
+        // When using the visco_plastic material model,
+        // make sure that use_fixed_elastic_timestep == false
+        // and stabilization_time_scale_factor == 1, because
+        // the current implementation does not support
+        // an elastic timestep that is not equal to the computational
+        // timepstep. For the same reasons, make sure that stress
+        // averaging is turned off and no damping is applied.
+        // If a damper is used, the current implementation in visco_plastic
+        // is not correct. The way the effective viscosity is computed is only valid
+        // in the absence of a damper and for a yield stress that can be expressed as
+        // some value of the second invariant of the deviatoric stress.
```

Comment:
Is this solved by simply taking the harmonic average of the damper and viscoelastic viscosity?

Reply:
No, at the moment we don't know how the damper could be correctly included.